website/docs/docs/build/metricflow-commands.md

```diff
@@ -338,13 +338,13 @@ mf query --metrics order_total,users_active --group-by metric_time # In dbt Core
 
 <TabItem value="eg2" label="Dimensions">
 
-You can include multiple dimensions in a query. For example, you can group by the `is_food_order` dimension to confirm if orders were for food or not. 
+You can include multiple dimensions in a query. For example, you can group by the `is_food_order` dimension to confirm if orders were for food or not.  Note that when you query a dimension, you need to specify the primary entity for that dimension.
 
 **Query**
 ```bash
-dbt sl query --metrics order_total --group-by metric_time,is_food_order # In dbt Cloud
+dbt sl query --metrics order_total --group-by primary_entity__is_food_order # In dbt Cloud
```

Comment:
Let's use `order_id__is_food_order` instead of `primary_entity__is_food_order` since that's what is used in teh `where clause` tab